ganga-data.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: More Ganga
+minutes: 10
+---
+> ## Learning Objectives {.objectives}
+>
+> * Set the input data with BKQuery
+> * Use LHCbDatasets
+> * Set the location of the output of our jobs
+> * Set the location of your .gangadir
+> * Access output stored on the grid
+
+The input data can be specified for your job with the BKQuery tool. The path for the data can be found using the online Dirac portal and passed to the `BKQuery` to get the dataset. For example, to run over the Stripping 21 MagUp, Semileptonic stream
+
+```python
+Ganga In [3]: j.inputdata = BKQuery('/LHCb/Collision12/Beam4000GeV-VeloClosed-MagUp/Real Data/Reco14/Stripping21r0p1a/90000000/SEMILEPTONIC.DST').getDataset()
+Ganga In [4]: j.inputdata
+Ganga Out [4]: 
+ LHCbDataset (
+   depth = 0,
+   treat_as_inputfiles = False,
+   persistency = None,
+   files = [3717 Entries of type 'DiracFile'] ,
+   XMLCatalogueSlice =    LocalFile (
+     namePattern = ,
+     compressed = False,
+     localDir = 
+   ) 
+ )
+```
+This is a list of `DiracFile`, the Ganga object for files stored on the grid. We can access one locally via the `accessURL`:
+```python
+Ganga In [5]: j.inputdata[0].accessURL()
+Ganga Out [5]: ['root://bw32-4.grid.sara.nl:1094/pnfs/grid.sara.nl/data/lhcb/LHCb/Collision12/SEMILEPTONIC.DST/00051179/0000/00051179_00006978_1.semileptonic.dst']
+```
+The returned path can be used by Bender to explore the contents of the DST.
```

Comment:
Could mention the DST explorer lesson too, which I assume will also work with these URLs.